### ISSUE TITLE
as an operator of RAMS, I'd like it to tell me when email categories are queued and need approval

### ISSUE THREAD

**Issue by binary1230** [magfest/ubersystem#2431]
The system currently queues emails that need approval, but there's no way to find out about it unless you specifically check the page.  In the past this has made us miss emails that were supposed to go out on a certain date, but no one remembered to check (we have a TON of email categories so it's easy to miss).

A simple version first pass version (probably all that's needed) of this would run a daemon task once per day which looks for queued emails and sends a report to the staffing ops dept, and then those folks can follow up with the dept heads to get email text approved.

A more complex version could send the email reminders to approve emails to the department heads directly, or to slack or something like that.

P.S. I thought we had this issue logged already but apparently not.  I would consider this a high priority issue because it will really help automate workflows for departments like STOPs that need to keep track of and babysit a lot of emails that ubersystem sends out.

**Comment by kitsuta** [comment]
I would actually say that it should report email categories only once when they get approved, and to send it to the email sender (e.g. if the email should come from regsupport@, send the notification to regsupport@). Otherwise, yeah, +1 on this issue

**Comment by binary1230** [comment]
is what you're saying then we should have 
1) a nightly report that goes to stops (or whatever) if there are queued emails in a category
2) an email that goes to the email from address (like music@event.com for band emails) once an email has been approved that says something like 'we just approved this email and X copies of it are going out, here's some sample text'

?

**Comment by kitsuta** [comment]
No, sorry, I mistyped at first. I think there should be:
1. An email that goes to the email "from" address once an email has queued emails in a category
2. Nothing else, no nightly reports, I think those are spammy

**Comment by binary1230** [comment]
i............would still be into the nightly reports, as someone who gets kept up at night thinking 'did we remember to approve that email'?

but maybe could send to a less-used address than STOPs.  or weekly reports? every other day?

**Comment by RobRuana** [comment]
Also for what it's worth, we do have a lot of unused space on the admin landing page. We could absolutely show a list of pending emails that need approval in that space. This won't help notify people that they need to log in to ubersystem. But at least it will put the work they need to do right up front.

![screen shot 2017-07-06 at 10 52 59 am](https://user-images.githubusercontent.com/2592431/27917237-71719af8-6239-11e7-8768-785c0f38a411.png)

**Comment by RobRuana** [comment]
Was the pending email report created by @binary1230 enough to close this issue?

**Comment by binary1230** [comment]
we could close this, though Vicki has a couple of suggestions that may make sense to implement in the future.  particularly, sending the pending email report directly to the depts that are affected (instead of all of them to STOPs)

I would at minimum mark the remainder of this issue as low priority, we have about 90% of the functionality in and working now

**Comment by kitsuta** [comment]
The idea I had is pretty non-trivial to implement, so in the interest of priorities, I'm taking this off the pre-reg epic.

**Comment by kitsuta** [comment]
New idea for this:
1. Organize emails by the department that sends them
2. Include a step in the department head checklist for any departments with pending emails to review + approve them

**Comment by kitsuta** [comment]
Fixed by #2972. @binary1230 comments on PRs don't auto-close issues; remember you can edit a description of a PR if you get the number wrong the first time.

**Comment by RobRuana** [comment]
Nice catch!
